### PR TITLE
Simplify dev setup instructions

### PR DESCRIPTION
**Pull request by inulty-dfe**
### Context

Simplify developer setup instructions.

### Changes proposed in this pull request

Encourage developers to use `./bin/setup` in order to initialise the application locally

### Checklist

- [ ] Make sure all information from the Trello card is in here
- [ ] Attach to Trello card
- [ ] Rebased main
- [ ] Cleaned commit history
- [ ] Tested by running locally
- [ ] Inform data insights team due to database changes

### CODE REVIEW

**Review by avinhurry — APPROVED** [review]

File: bin/setup [hunk 1]
```diff
@@ -14,20 +14,26 @@ FileUtils.chdir APP_ROOT do
...
+  system("yarn install")
   system! "gem install bundler --conservative"
   system("bundle check") || system!("bundle install")
 
-  # puts "\n== Copying sample files =="
-  # unless File.exist?("config/database.yml")
-  #   FileUtils.cp "config/database.yml.sample", "config/database.yml"
-  # end
+  puts "\n== Copying development settings file =="
+  system! "sed 's/secret: secret/secret: persona/' config/settings/development.yml > config/settings/development.local.yml"
```
Comment (avinhurry) [inline bin/setup:22]: Should we consider users on non unix-like operating systems? 🤔

Comment (inulty-dfe) [reply bin/setup:22]: In theory, yes!

I'll update to print a warning if on Windows. I can't test a sed-like command and it isn't worth doing either.

**Comment by inulty-dfe** [comment]
The sed warning for windows users looks like this.

Screenshot from 2023-06-30 09-33-51

**Comment by avinhurry** [comment]
> The sed warning for windows users looks like this.
> 
> Screenshot from 2023-06-30 09-33-51

@inulty-dfe I think it should be:

```authentication:
  secret: secret
  mode: persona
  ```
  
  and not 
  ```authentication:
  secret: secret
 secret: persona
  ```
  
  Maybe we need to add `mode: persona` to `development.yml`?

**Pull request merged**